src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/Public.Data/PreviewTokenCreateRequest.cs

```diff
@@ -8,17 +8,49 @@ public record PreviewTokenCreateRequest
     public required Guid DataSetVersionId { get; init; }
 
     public required string Label { get; init; }
-
+    
+    public DateTimeOffset? Activates { get; init; }
+    
+    public DateTimeOffset? Expires { get; init; }
+    
     public class Validator : AbstractValidator<PreviewTokenCreateRequest>
     {
         public Validator()
         {
+            When(request => request.Activates.HasValue && request.Expires.HasValue, () =>
```

Comment:
I don't understand why this check `When(request => request.Activates.HasValue && request.Expires.HasValue, () =>`  is necessary. Surely it means we only validate if both are set?

One can be set without the other, in which case it still needs validating.